CONTRIBUTING.md

```diff
@@ -1,35 +1,226 @@
-# Filing an issue
+# How to contribute to qBittorrent
 
-### Must read
-* If you aren't sure, you can ask on the [**forum**](http://forum.qbittorrent.org) or read our [**wiki**](http://wiki.qbittorrent.org) first.
-* Do a quick **search**. Others might already reported the issue.
-* Write in **English**!
-* Provide **version** information: (You can find version numbers at menu `Help -> About -> Libraries`)
-  ```
-  qBittorrent:
-  Qt:
-  libtorrent:
-  boost:
-  OS version:
-  ```
-* Provide **steps** to reproduce the problem, it will be easier to pinpoint the fault.
-* **Screenshots**! A screenshot is worth a thousand words. just upload it. [(How?)](https://help.github.com/articles/file-attachments-on-issues-and-pull-requests)
+There are three main ways to contribute to the project.
+Read the respective section to find out more.
 
-### Good to know
-* **Be patient**. The dev team is small and resource limited. Devs finding their free time, analyzing the problem and fixing the issue, it all takes time. :clock3:
-* If you can code, why not become a **contributor** by fixing the issue and open a pull request? :wink:
-* Harsh words or threats won't help your situation. What's worse, your complain will (very likely) to be **ignored**. :fearful:
+### Table Of Contents
+
+*   **[Bug reporting etiquette](#bug-reporting-etiquette)**
+
+
```

Comment:
Please change double empty lines to one empty line and check other places below.

IMO you are free to squash the commits.

Reply:
I was able to remove some empty lines without affecting the formatting. The others are necessary to maintain the desired layout; see the images below:
![a](https://user-images.githubusercontent.com/17580742/38270009-0608a3ee-377a-11e8-8a61-285bf4c9f8f7.png)
![b](https://user-images.githubusercontent.com/17580742/38270013-07e938f4-377a-11e8-8306-aa7428d885c7.png)

Reply:
@Chocobo1 
Is it ok now?

Reply:
@FranciscoPombal:  
IIRC, the standard Markdown format does not technically support extended gaps between paragraphs at all.  

Could you please test rendering this file in a different Markdown interpreter, and post similar screenshots of that one as well?

Reply:
>The others are necessary to maintain the desired layout

Just FYI, see preview on github: https://github.com/FranciscoPombal/qBittorrent/blob/02fda4ccd4ebd2fb9c721f4b092a512927bca7b3/CONTRIBUTING.md#table-of-contents